crates/anvil-polkadot/src/api_server/error.rs

```diff
@@ -28,12 +30,6 @@ pub enum Error {
     InternalError(String),
 }
 
-impl From<subxt::Error> for Error {
-    fn from(err: subxt::Error) -> Self {
-        Self::ReviveRpc(EthRpcError::ClientError(err.into()))
-    }
-}
```

Comment:
Added this back since I also removed the `Subxt` variant. No need to use it, since as you mentioned, we can rely on the catch all internal error variant.